.phpstan/baseline.neon

```diff
@@ -3,3 +3,16 @@ parameters:
         - message: '#^Access to undefined constant static\(Liip\\ImagineBundle\\Config\\Filter\\Type\\FilterAbstract\)::NAME\.$#'
           paths:
               - ../src/Config/Filter/Type/FilterAbstract.php
+
+        - message: "#^Call to an undefined method Symfony\\\\Component\\\\Config\\\\Definition\\\\Builder\\\\NodeParentInterface\\:\\:validate\\(\\)\\.$#"
+          count: 1
+          path: ../src/DependencyInjection/Configuration.php
+
+        # BC Layer for Symfony < 5.1
+        - message: "#^If condition is always true\\.$#"
+          count: 1
+          path: ../src/Controller/ImagineController.php
+
+        - message: "#^Unreachable statement \\- code above always terminates\\.$#"
```

Comment:
is this a mistake of phpstan or of us? or some BC layer?

Reply:
Is the same BC as above, I've added a comment.

https://github.com/liip/LiipImagineBundle/blob/56a72358b3338a22d59bdbcb4a4c57221fa4a524/src/Controller/ImagineController.php#L110-L126

The `if` is always true with the version installed.

Reply:
thanks!

hm, maybe phpstan should know about `version_compare` - it is kind of obvious that this is done for a reason i'd say...

ah, i think i found the solution: https://github.com/phpstan/phpstan/issues/2636 - we should list the Kernel::VERSION constant in dynamicConstantNames

Reply:
nice, thanks! didn't know about that one, I had to change one thing, I'll add the comment there.

Reply:
me neither, but i thought this must be a common issue and had a good guess what to search for :-D